Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/FixedWidthFlatFile.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Frends.FixedWidthFlatFile.ConvertToJSON.Definitions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+#pragma warning disable 1591
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON
+{
+    public static class FixedWidthFlatFile
+    {
+        /// <summary>
+        /// This is a task that converts given value to json string.
+        /// Documentation: https://github.com/CommunityHiQ/Frends.FixedWidthFlatFile.ConvertToJSON
+        /// </summary>
+        /// <param name="input">What value to convert.</param>
+        /// <returns>{string Data} </returns>
```

Comment:
Please modify the return documentation so that it will clearly state that an object is returned to avoid causing confusion to users. So `object { string Data }`.